### ISSUE TITLE
"az iot hub list" returns "resourceGroup" property twice in output json

### ISSUE THREAD

**Issue by markusstuhler** [Azure/azure-iot-cli-extension#70]
The command "az iot hub list" returns the property "resourceGroup" twice (with different casing):

```
[
  {
    ...
    "resourceGroup": "testhub",
    "resourcegroup": "testhub",
    ..
  }
]
```

That's valid JSON, however, it's hard to parse this with PowerShells "ConvertFrom-Json" as PowerShell is case insensitive and throws an error. 

Perhaps you could evaluate if this is intended or required to be like this.

**Comment by digimaun** [comment]
Thanks for reporting the issue @markusstuhler. This command you are mentioning lives in the base CLI (and not the extension) but regardless the issue should be looked at.

@iluican  @anusapan

**Comment by anusapan** [comment]
This issue was related to CLI Core and it has been fixed into CLI Core. The impact will come into the act from the next release.

**Comment by digimaun** [comment]
Fix should be in.
